pkg/scheduler/framework/v1alpha1/waiting_pods_map.go

```diff
@@ -21,7 +21,7 @@ import (
 	"sync"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
No need to change here?

Reply:
This won't be a problem IMO. This was automatically introduced by goimports, and it is not worth to fight the tools.